.github/workflows/hil_sample_zephyr.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           pip3 install -r zephyr/scripts/requirements-build-test.txt
           pip3 install -r zephyr/scripts/requirements-run-test.txt
 
-          pip3 install git+https://github.com/golioth/python-golioth-tools@v0.2.0
+          pip3 install git+https://github.com/golioth/python-golioth-tools@7fad3fe3216a15d42fc9a35f50c434b9b8eba062
```

Comment:
This ref should be changed to `v0.3.0` after https://github.com/golioth/python-golioth-tools/pull/7 is merged and the merge commit is tagged.